rom/laghos.cpp

```diff
@@ -1727,8 +1822,81 @@ int main(int argc, char *argv[])
         }
     }
 
+    double errorIndicator = INT_MAX;
+    bool errorIndicatorComputed = false;
+    int errorIndicatorVecSize = 0;
+
     if (rom_online)
     {
+
+        // If using the greedy algorithm, calculate the error indicator using the FOM lifted during
+        // the second to last step compared against the FOM lifted at the last step.
```

Comment:
Is this comment meant for the `useLastLiftedSolution` case?

Reply:
Yes, I'll clarify.